src/compiler/reviews.md

```diff
@@ -348,76 +270,63 @@ r? bootstrap
 ```
 
 ### bors
+We never merge PRs directly. Instead, we use bors. A qualified reviewer with bors privileges
+(e.g., a [compiler team member](./membership.md)) will leave a comment like `@bors r+`. This
+indicates that they approve the PR.
 
-We never merge PRs directly. Instead, we use bors. A qualified reviewer with
-bors privileges (e.g., a [compiler team member](./membership.md)) will leave a
-comment like `@bors r+`. This indicates that they approve the PR.
-
-People with bors privileges may also leave a `@bors r=username` command. This
-indicates that the PR was already approved by `@username`. This is commonly done
-after rebasing.
+People with bors privileges may also leave a `@bors r=username` command. This indicates that the
+PR was already approved by `@username`. This is commonly done after rebasing.
 
 Finally, in some cases, PRs can be "delegated" by writing `@bors delegate+` or
-`@bors delegate=username`. This will allow the PR author or the delegated user
-to approve the PR by issuing `@bors` commands like the ones above (but this
-privilege is limited to the single PR).
+`@bors delegate=username`. This will allow the PR author or the delegated user to approve the PR
+by issuing `@bors` commands like the ones above (but this privilege is limited to the single PR).
```

Comment:
Suggestion:

```suggestion
`@bors delegate=username`.

- This will allow the PR author or the delegated user to approve the PR
by issuing `@bors` commands like the ones above or run try jobs (but this privilege is limited to the single PR).
- This should *only* be done if you are reasonably confident the PR author acts in good faith and won't introduce malicious code into try-build or full-build artifacts.
```

Sometimes I leave a comment like

```
@bors delegate+ (for try-jobs only)
```

to indicate scope of delegation, i.e. only try builds but not r+.

Reply:
Fixed!